.gitlab-ci.yml

```diff
@@ -725,7 +725,7 @@ workflow:
   - <<: *if_run_all_kmt_tests
   - changes:
       paths: *security_agent_change_paths
-      compare_to: main # TODO: use a variable, when this is supported https://gitlab.com/gitlab-org/gitlab/-/issues/369916
+      compare_to: 7.61.x # TODO: use a variable, when this is supported https://gitlab.com/gitlab-org/gitlab/-/issues/369916
```

Comment:
FWIW the ticket is now closed and this feature appears to be supported in gitlab 17.x (we are still on 16.x)

Reply:
Thanks! I'll let the devx infra team know!